x11-misc/sunflower/sunflower-0.3_alpha61.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit gnome2-utils python-r1 xdg-utils
+
+MY_PN="Sunflower"
+DESCRIPTION="Small and highly customizable twin-panel file manager with plugin-support"
+HOMEPAGE="https://github.com/MeanEYE/Sunflower
+	http://sunflower-fm.org/"
+SRC_URI="http://sunflower-fm.org/pub/sunflower-${PV/_alpha/-}.tgz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}
+	>=dev-python/pygtk-2.15.0[${PYTHON_USEDEP}]
+	>=dev-python/notify-python-0.1[${PYTHON_USEDEP}]
+	dev-python/chardet[${PYTHON_USEDEP}]
+	gnome-base/librsvg:2"
+
+S=${WORKDIR}/${MY_PN}
+
+src_prepare() {
+	default
+	find "${S}" -name "*.py[co]" -delete || die
```

Comment:
`${S}` is the current directory so it can be omitted (or replaced by `.` when there's no path and you want to be compatible with FreeBSD).